packages/node_modules/@ciscospark/widget-roster/src/components/add-participant/index.js

```diff
@@ -27,6 +29,16 @@ export default function AddParticipant(props) {
     props.onAddPerson(person);
   }
 
+  function handleInvite() {
+    props.onAddPerson(props.searchTerm);
+  }
+
+  function handleKeyPressInvite(e) {
+    if (e.key === `Enter` || e.key === ` `) {
```

Comment:
Does this add the person when spacebar is pressed? Doesn't spacebar usually page down?

Reply:
It does when the person is selected and spacebar is pressed. According to a11y standards, buttons allow enter or space to select.